### PR TITLE
docs: update style and font for docs

### PR DESCRIPTION
**Pull request by fubuloubu**
### What I did
from @agBabbbyCakes

### CODE REVIEW

**Comment by fubuloubu** [comment]
- [ ] Someone just needs to verify the updates to style on the website built using this branch

**Review by agBabbbyCakes — COMMENTED** [review]
Please @fubuloubu check this out - ready to go baby - all i did was get that purp

**Comment by unparalleled-js** [comment]
I tried it out. I really love this font!

But in my opinion, the text is now a little too small, except for the main header-  that is too large.

Screenshot 2023-01-20 at 18 11 02

**Comment by unparalleled-js** [comment]
Also the highlight color seems a little dark to me:

Screenshot 2023-01-20 at 18 12 33

**Review by unparalleled-js — DISMISSED** [review]
Ultimately, I think @agBabbbyCakes can make the call here, but I added a couple thoughts.

**Comment by fubuloubu** [comment]
> Ultimately, I think @agBabbbyCakes can make the call here, but I added a couple thoughts.

From what I can see, needs a bit more work

**Comment by agBabbbyCakes** [comment]
aye! thank you for the feedback @fubuloubu @unparalleled-js 

This was basically just a test run to see if the changes stuck - still working on it and will have an update super soon!

Will consider the text size + highlight color

**Comment by agBabbbyCakes** [comment]
[](url)

**Review by agBabbbyCakes — COMMENTED** [review]
made a bunch of highlight changes + font sizes + changed the color scheme on the links

File: docs/_static/custom.css [hunk 1]
```diff
@@ -194,21 +208,128 @@ footer {
...
+@media only screen and (min-device-width: 320px) and (max-device-width: 480px) {
+
+a {
+  color:rgb(93, 0, 255);
+  cursor: pointer;
+}
+
+}
+
+/*NEW AG ADDITIONS * EVEN NEWERE - TRYING TO MESS WITH THT SIDEBAR
```
Comment (unparalleled-js) [inline docs/_static/custom.css:237]: Some these comments are kind of distracting. Why all caps and alternative spellings?

It is very characteristic of you, but I am not it is necessary in this document.

**Review by unparalleled-js — DISMISSED** [review]
This is beautiful. Wow, great work. I actually have the urge to browse the docs, it is odd, how did you do that? I feel like I am reading documentation on a pristine marble slab.

Unrelated to this PR, mobile still looks funny on the docs site for some reason.

Also do you care if we adjust some of these comments? See comment.

File: docs/_static/custom.css (hunk 1 above)
Comment (fubuloubu) [reply docs/_static/custom.css:237]: Yeah let's clean these out :heart:

**Review by agBabbbyCakes — COMMENTED** [review]
fixed up the css comments

File: docs/_static/custom.css [hunk 2]
```diff
@@ -194,21 +208,121 @@ footer {
...
+}
+
+.wy-side-nav-search input[type=text] {
+  width: 80%;
+  border-radius: 25px;
+  padding: 6px 12px;
+  border-color: #000000;
+  border: width 1.5em;
+}
+
```
Comment (fubuloubu) [inline docs/_static/custom.css:310]:
```suggestion
```

File: docs/_static/custom.css [hunk 3]
```diff
@@ -194,21 +208,121 @@ footer {
...
+.logo {
+  filter: invert(.85);
+  width:250px!important;
+  height:auto;
+  position: relative;
+  top:20!important;
+  left:20!important;
+margin-left:-10px;  z-index: 240;
```
Comment (fubuloubu) [inline docs/_static/custom.css:290]:
```suggestion
  margin-left:-10px;  z-index: 240;
```

File: docs/_static/custom.css [hunk 4]
```diff
@@ -194,21 +208,121 @@ footer {
...
+    font-size: 100%;
+    color: #000000;
+}
+
+.wy-menu-vertical a:hover {
+  line-height: 28px;
+  font-size:18px;
+  font-weight: 400;
+  font-family: 'Barlow', sans-serif;
+  /*padding: 0.4045em 1.618em;*/
```
Comment (fubuloubu) [inline docs/_static/custom.css:274]:
```suggestion
```

File: docs/_static/custom.css [hunk 5]
```diff
@@ -194,21 +208,121 @@ footer {
...
+  .wy-menu-vertical a {
+    line-height: 28px;
+    font-size:18px;
+    font-weight: 400;
+    font-family: 'Barlow', sans-serif;
+    padding: 0.4045em 1.618em;
+    display: block;
+    position: relative;
+    font-size: 100%;
+    color: #000000;
```
Comment (fubuloubu) [inline docs/_static/custom.css:266]:
```suggestion
.wy-menu-vertical a {
  line-height: 28px;
  font-size:18px;
  font-weight: 400;
  font-family: 'Barlow', sans-serif;
  padding: 0.4045em 1.618em;
  display: block;
  position: relative;
  font-size: 100%;
  color: #000000;
```

File: docs/_static/custom.css [hunk 6]
```diff
@@ -194,21 +208,121 @@ footer {
...
+  padding:0px!important;
+}
+
+.reference {
+  color:rgb(93, 0, 255);
+}
+
+.wy-menu-vertical li.current a {
+  background: #ff969600;
   }
```
Comment (fubuloubu) [inline docs/_static/custom.css:255]:
```suggestion
}
```

File: docs/_static/custom.css [hunk 7]
```diff
@@ -194,21 +208,121 @@ footer {
...
-  a:hover {
-    color: rgb(0, 0, 0);
+
+/*AG LINKS */
+a {
+  color:rgb(93, 0, 255);
+  cursor: pointer;
+  
+}
+
```
Comment (fubuloubu) [inline docs/_static/custom.css:226]:
```suggestion
```

File: docs/_static/custom.css [hunk 8]
```diff
@@ -194,21 +208,121 @@ footer {
...
+}
 
-  a:visited {
-    color: #c2006b;
-  }
 
-  a:hover {
-    color: rgb(0, 0, 0);
+
+/*AG LINKS */
```
Comment (fubuloubu) [inline docs/_static/custom.css:220]:
```suggestion
```

File: docs/_static/custom.css [hunk 9]
```diff
@@ -1,90 +1,104 @@
...
 h3 {
   font-size: 24px;
   line-height: 24px;
-  font-family: "Space Mono", sans-serif;
+  font-family: 'Barlow', sans-serif;
   letter-spacing: -0.05em;
   font-weight: 300;
+  color: #000000;
+  
 }
```
Comment (fubuloubu) [inline docs/_static/custom.css:65]:
```suggestion
```

File: docs/_static/custom.css [hunk 10]
```diff
@@ -1,90 +1,104 @@
...
 
-a {
-  color: #e70000;
-  cursor: pointer;
-}
-
-a:visited {
-  color: #e70000;
-}
```
Comment (fubuloubu) [inline docs/_static/custom.css:66]:
```suggestion
```

File: docs/_static/custom.css [hunk 11]
```diff
@@ -1,90 +1,104 @@
-@import url("https://fonts.googleapis.com/css?family=Space+Mono&display=swap");
-@import url("https://fonts.googleapis.com/css?family=Rubik&display=swap");
+@import url('https://fonts.cdnfonts.com/css/barlow'); 
+
+
```
Comment (fubuloubu) [inline docs/_static/custom.css:3]:
```suggestion
```

**Review by fubuloubu — COMMENTED** [review]
Cleanup

**Comment by fubuloubu** [comment]
@agBabbbyCakes any more changes to be made?

@unparalleled-js we lookin' good to merge?

**Review by unparalleled-js — APPROVED** [review]
Looks good!
The only thing I added was running `prettier docs/_static/custom.css -w` so the document is nicely formatted.

**Pull request merged**